client/src/component/Title/Title.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import 'font-awesome/css/font-awesome.min.css'
+import './Title.css'
+
+const Title = () => {
```

Comment:
```suggestion
class Title extends React.Component
```

Convert Method into a React Component